src/zerocopy.rs

```diff
@@ -75,6 +100,16 @@ impl<T> Receiver<T>
 where
     T: OwningMessage,
 {
+    /// Receives a non-owning message from this channel.
+    ///
+    /// If there is a queued owning messaged, that message is immediately
+    /// popped. Otherwise, this function waits for the sender to send
+    /// non-owning data, which may be used directly.
+    ///
+    /// In order to preserve the safety of non-owning messages, the sender
+    /// cannot finish sending a non-owning message until this function's
+    /// closure has finished executing. **Exit the closure as quickly and as
+    /// quickly as possible to avoid timing attack vulnerabilities.**
```

Comment:
This is probably a typo, I'd imagine that an adverb other than "quickly" was intended to be used in the first instance?

Reply:
Nice catch, thank you! I meant to say "consistently", I think.

Reply:
Oh yeah that'd make sense. So `as quickly and consistently as possible`, then? The consistency should probably be emphasized over the speed, since that's where the information disclosure issues really come from.